packages/gestalt/src/SlimBanner.js

```diff
@@ -116,18 +116,18 @@ type Props = {|
    */
   dismissButton?: DismissButtonType,
   /**
-   * Helper [Link](https://gestalt.pinterest.systems/web/link) to be placed after the message. See the [helperLink variant](https://gestalt.pinterest.systems/web/slimbanner#helperLink) to learn more.
+   * Helper [Link](https://gestalt.pinterest.systems/web/link) to be placed after the message. See the [Message variant](https://gestalt.pinterest.systems/web/slimbanner#Message) to learn more.
    */
   helperLink?: HelperLinkType,
   /**
    * Label to describe the status icon’s purpose. See the [Accessibility guidelines](https://gestalt.pinterest.systems/web/slimbanner#Accessibility) for details on proper usage.
    */
   iconAccessibilityLabel?: string,
   /**
-   * Main content of SlimBanner. Content should be [localized](https://gestalt.pinterest.systems/web/slimbanner#Localization).
+   * Main content of SlimBanner. Content should be [localized](https://gestalt.pinterest.systems/web/slimbanner#Localization). See the [Message variant](https://gestalt.pinterest.systems/web/slimbanner#Message) to learn more.
    *
    */
-  message: string,
+  message: string | Element<typeof Text>,
```

Comment:
![Screen Shot 2022-11-09 at 5 06 45 PM](https://user-images.githubusercontent.com/12059539/200975713-216ace98-078c-4ca8-bac3-e19b0f40d698.png)

We need to make sure this shows `React.Element`. You'll need to adjust the logic in [GeneratedPropTable](https://github.com/pinterest/gestalt/blob/d30cd30ba74efd4b1e6b8ed02abf71f6293bbe9d/docs/docs-components/GeneratedPropTable.js#L101)…since the current regex is checking for "Element" at the beginning of a string (to avoid matching "HTMLDivElement"), you may need to add a new `.replace` call that looks for "Element" with preceding whitespace

Reply:
I don't see any changes to GeneratedPropTable on this PR? Are you making that change in a separate PR?

Reply:
DONE! Sorry I changed the wrong prop name ahahah